package pl.dkobylarz.signlearning.domain.authorization.exception

class UsernameAlreadyExistsException: RuntimeException("Username already exists!")
